doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -324,9 +320,9 @@ described elsewhere
 
    .. example:: :cmd:`SearchPattern` examples
 
-      .. coqtop:: in
+      .. coqtop:: none
 
-         Require Import Arith.
+         Axiom add_comm: forall n m, n + m = m + n.
```

Comment:
The `SearchPattern` with lists below should be changed to something that gives some output.